src/main/java/com/taskadapter/redmineapi/bean/Project.java

```diff
@@ -46,6 +47,13 @@
      * Trackers available for this project
      */
     public final static Property<Set<Tracker>> TRACKERS = (Property<Set<Tracker>>) new Property(Set.class, "trackers");
+    
+    /**
+     * Project status values, as defined in http://www.redmine.org/projects/redmine/repository/entry/trunk/app/models/project.rb
+     */
+    public static final Integer STATUS_ACTIVE = 1;
+    public static final Integer STATUS_CLOSED = 5;
+    public static final Integer STATUS_ARCHIVED= 9;
```

Comment:
I prefer to avoid hardcoding these values. at most we can add a new file with constants, like Redmine318Constants with values that are known to work with Redmine 3.1.8, or something like that.
and then let users of the library decide which version of constants to use in their code.

hardcoding these values in the main library code itself could make the library invalid at some point in the future, potentially for a long time until someone decides to fix this, and even then - if values become different, do you want to support both versions in the code? 

having these values outside of the main library code is cheaper to maintain. let's remove these

Reply:
ok, removing these lines